### ISSUE TITLE
bug in removing self connections

### ISSUE THREAD

**Issue by jajcayn** [neurolib-dev/neurolib#60]
the piece of code in `loadDefaultParams.py` for various models which is supposed to get rid of self connections:
```python
params.Cmat = Cmat.copy()  # coupling matrix
np.fill_diagonal(Cmat, 0)  # no self connections
```
is not working, since the second line fills diagonal in the original `Cmat` variable, and not inside `params.Cmat`

**Comment by jajcayn** [comment]
also, the function description states that ":param Cmat: Structural connectivity matrix (adjacency matrix) of coupling strengths, will be normalized to 1. If not given, then a single node simulation will be assumed, defaults to None" but there is no normalisation to 1 happening later in the code. the matrix is passed into the integrator as is

**Comment by caglorithm** [comment]
Closed by #61
